Add map of US States & Territories

fatima3558 opened the pull request.

Commit:
add us states and territories svg

Commit:
add package.json and index.js

Commit:
add LICENSE & README; delete comments

Commit:
add usa-states-territories to main README

VictorCazanave commented on packages/usa-states-territories/package.json:
```diff
@@ -0,0 +1,20 @@
+{
+	"name": "@svg-maps/[new-map]",
```
Could you replace the name here please?

VictorCazanave commented on packages/usa-states-territories/usa-states-territories.svg:
```diff
@@ -0,0 +1,287 @@
+	aria-label="Map of United States and Territories"
+>
+	<path
+		id="AK"
```
Could you use lowercase ids please?

VictorCazanave commented on packages/usa-states-territories/usa-states-territories.svg:
```diff
@@ -0,0 +1,287 @@
+		id="DC"
+		name="Washington, DC"
+		d="M801.8,253.8 l-1.1-1.6 -1-0.8 1.1-1.6 2.2,1.5z"/>
+	<circle id="DC2" stroke="#FFFFFF" stroke-width="1.5" cx="801.3" cy="251.8" r="5" opacity="1"/>
```
Could you delete this tag please?

VictorCazanave reviewed (commented):
Thanks a lot for your contribution @fatima3558! 😃

Just few things to fix, then I'll glad to merge it.

Commit:
rename files; remove circle tag; lowercase ids

Commit:
re-generate usa.states-territories/index.js

Commit:
run
up to date, audited 831 packages in 1s

39 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities in attempt to sync package-lock.json

Commit:
revert package-lock.json

Commit:
Merge branch 'master' into master

fatima3558 commented:
had a little trouble there with `package-lock.json`, but I think I got it squared away. thanks for taking a look, let me know if any additional changes are needed.

VictorCazanave commented:
Thanks again!

@all-contributors please add @fatima3558 for code
